agora/footfall_ai_api/src/video_capture.py

```diff
@@ -35,17 +34,19 @@ def _reader(self):
                 if not self.running:
                     return
             
-           # Save the frame if it is at the specified interval
+            # Save the frame if it is at the specified interval
             if self.frame_count % self.frame_interval == 0:
                 self.q.put(cv2.resize(frame, (640, 360)))
```

Comment:
What about having two env variables, FRAME_HEIGH and FRAME_WIDTH and just resize it here to the desired size? Maybe better than doing two resigning, first to 640 *320 and then the other one you do as part of processor